Libplanet.Tools/download.mjs

```diff
@@ -0,0 +1,145 @@
+// The planet binary downloader for npm.
+// Inspired by Elm's npm packaging: <https://www.npmjs.com/package/elm>.
+"use strict";
+const nodeMajorVersion = process.versions.node.split('.')[0];
+if (nodeMajorVersion < 12) {
+  throw new Error("Requires Node.js v12 or Higher Version.");
+}
```

Comment:
You may use [`engines`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines) field for specifying the required Node.js version range. This will warn users when installing on an unsupported Node.js version.

Reply:
Applied: 923770e7793b962fda167e215c3b216c8f420e3f.